Python 3.13.1 (tags/v3.13.1:0671451, Dec  3 2024, 19:06:28) [MSC v.1942 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
range()
Traceback (most recent call last):
  File "<pyshell#0>", line 1, in <module>
    range()
TypeError: range expected at least 1 argument, got 0
range(1)
range(0, 1)
print(range(1,6))
range(1, 6)
range(1,6)
range(1, 6)
a=ranhe(1,6)
Traceback (most recent call last):
  File "<pyshell#4>", line 1, in <module>
    a=ranhe(1,6)
NameError: name 'ranhe' is not defined. Did you mean: 'range'?
a=range(1,6)
print(a)
range(1, 6)
a=tuple(range(1,6))
print(a)
(1, 2, 3, 4, 5)
a=set(range(6,1,-1))
print(type(a))
<class 'set'>
<class 'set'>
SyntaxError: invalid syntax
a=set(range(6,1,-1))
print(a)
{2, 3, 4, 5, 6}
a=list(range(6,1,-1))
print(a)
[6, 5, 4, 3, 2]
[6, 5, 4, 3, 2]
[6, 5, 4, 3, 2]
print(range(,1,6,9,3))
SyntaxError: invalid syntax
SyntaxError: invalid syntax
SyntaxError: invalid syntax
print(str(set(list(tuple(range(6,0,-1))))))
{1, 2, 3, 4, 5, 6}

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
1
2
3
4
5
6
7
8
9

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================










==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
1 2 3 4 5 6 7 8 9 

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
112131415161718191

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
Traceback (most recent call last):
  File "C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py", line 23, in <module>
    print(i,end=1)
TypeError: end must be None or a string, not int

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
11
12
13
14
15

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
11 12 13 14 15 

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
15 12 9 6 3 

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
25
20
15
10
5

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
0
1
2
3
4

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
5
4
3
2

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
5

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
5
4
3
2
1

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
5
3
1

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
5
2

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
5
1

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
5
1
1
2
3
4
5

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
1
2
3
4
5

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
Traceback (most recent call last):
  File "C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py", line 109, in <module>
    for i in range(a):
TypeError: 'tuple' object cannot be interpreted as an integer

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
1
2
3
4

==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
m
a
n
s
i
>>> 
==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
Traceback (most recent call last):
  File "C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py", line 135, in <module>
    for i in range(a):
TypeError: 'str' object cannot be interpreted as an integer
>>> 
==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
0
1
2
3
4
>>> 4
4
>>> a="rajendra"
>>> len(a)
8
>>> 8
8
>>> a=mansi
Traceback (most recent call last):
  File "<pyshell#24>", line 1, in <module>
    a=mansi
NameError: name 'mansi' is not defined
>>> a="MANSI"
>>> LEN(A)
Traceback (most recent call last):
  File "<pyshell#26>", line 1, in <module>
    LEN(A)
NameError: name 'LEN' is not defined
>>> 
==================================== RESTART: C:/Users/Quastech/AppData/Local/Programs/Python/Python313/for loop 31-01-2025.py ====================================
6
5
4
3
2
